backend/src/main/java/de/sambam/gamenighttracker/service/GameService.java

```diff
@@ -0,0 +1,21 @@
+package de.sambam.gamenighttracker.service;
+
+import de.sambam.gamenighttracker.db.GameDb;
+import de.sambam.gamenighttracker.model.Game;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class GameService {
+
+    private final GameDb gameDb;
+
+    @Autowired
+    public GameService(GameDb gameDb) {
+        this.gameDb = gameDb;
+    }
+
+    public Game addNewGame(Game newGame) {
+        return gameDb.add(newGame);
```

Comment:
möglicherweise Sinnvoll hier ein Optional zu verwenden, falls das Spiel das angelegt werden soll bereits existiert und kein Spiel zurückgegeben werden kann?

Reply:
das baue ich erst, wenn die eigentlich Db angebunden ist. ^^

Reply:
when adding a game you should always return a game or throw. Therefor it is good not to return an optional ;) 
You should us an optional when you try to get a specific game.